cmd/prchecker/main.go

```diff
@@ -237,7 +250,7 @@ func PullRequestSpanningPathList(repository GitHubService, pullrequest int, path
 	return spanningChangesExist, changesets, nil
 }
 
-// IsChangesetSpanningPathList tests if a changeset is spanning
+// ChangesetSpanningPathList tests if a changeset is spanning
```

Comment:
- please make this a method on Changeset.
 - name it so it reads well in an `if` statement, e.g.

```
if a.equals(b) {}
if a.contains(b) {}
if changeSet.spansPaths(paths) {}
```

Reply:
Renamed to `isSpanningPaths()`

Reply:
OK, but fwiw, there are plenty of Go functions in the standard libs that return bool that don't begin with `is`;
https://golang.org/pkg/path/filepath/#HasPrefix
https://golang.org/pkg/strconv/#CanBackquote
https://golang.org/pkg/time/#Time.After
etc.